src/resources/postcss/tickets-refresh.pcss

```diff
@@ -445,14 +406,28 @@ p.description {
 #ticket_form .tooltip_container {
 	@mixin tribe-ticket-tooltip;
 
-	display: inline-block;
+	display: inline;
 	vertical-align: middle;
 
+	@media screen and ( max-width: 390px ) {
+		display: none;
+	}
+
 	.tooltip {
 		background-color: #efefef;
 		left: 100%;
 		margin: 0;
 		width: 200px;
+
+		@media screen and ( max-width: 782px ) {
+			-webkit-transform: none;
```

Comment:
Is autoprefixer not adding these? If not, IMHO it should go after the associated selector, so:
```
transform: none;
-webkit-transform: none;
```
Others may argue, but they should at least be grouped together? Food for conversation...